facebook-for-woocommerce.php

```diff
@@ -127,7 +129,7 @@ public function __wakeup() {
 	 */
 	public function init_plugin() {
 
-		if ( ! $this->plugins_compatible() ) {
+		if ( ! Checker::instance()->is_compatible( __FILE__, self::PLUGIN_VERSION ) || ! $this->plugins_compatible() ) {
```

Comment:
Shouldn't Checker::instance()->is_compatible( __FILE__, self::PLUGIN_VERSION ) move inside plugins_compatible ?

Reply:
This is a good catch, @rawdreeg. My approach was not to disturb existing checks. However, in this case, we can remove the `plugins_compatible` method and replace it with `Checker::instance()->is_compatible( FILE, self::PLUGIN_VERSION )` alone.

Reply:
I've implemented the above suggestion in this commit: https://github.com/woocommerce/facebook-for-woocommerce/pull/2640/commits/8eefaf340d5c5f310805b7b0dc07f9c40e3e46fc